### PR TITLE
WH1 Eligibility PDRSAug24 

### PR DESCRIPTION
**Pull request by SelvaRC**
All tests passing with storage volume

### CODE REVIEW

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml [hunk 1]
```diff
@@ -199,5 +210,129 @@
...
+        true,
+        false,
+        true,
+        true
+      ]
+  output:
+    WH1_F16_electric_PDRSAug24__installation_replacement_final_activity_eligibility:
+      [
+        true,
+        False,
```
Comment (alisha17) [inline openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml:333]: F is uppercase, can you change?

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py [hunk 2]
```diff
@@ -194,8 +172,9 @@ class WH1_F16_electric_PDRSAug24__equipment_certified_by_storage_volume(Variable
 
     def formula(buildings, period, parameters):
       storage_volume = buildings('WH1_F16_electric_PDRSAug24__storage_volume', period)
+      print("storage_volume",storage_volume)
       certified_AS_NZ_2712 = buildings('WH1_F16_electric_PDRSAug24__certified', period)
-
+      print("certified_AS_NZ_2712",certified_AS_NZ_2712)
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py:177]: remove print

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml [hunk 3]
```diff
@@ -199,5 +210,129 @@
...
+        true
+      ]
+    WH1_F16_electric_PDRSAug24__minimum_payment:
+      [ 
+        true,
+        true,
+        true,
+        true,
+        true,
+        true
```
Comment (alisha17) [inline openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml:273]: Hi @SelvaRC  sorry I think the true and false should have first letter uppercase. Please check other test files for the right format. And then can you run tests again?

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py [hunk 4]
```diff
@@ -194,8 +172,8 @@ class WH1_F16_electric_PDRSAug24__equipment_certified_by_storage_volume(Variable
 
     def formula(buildings, period, parameters):
       storage_volume = buildings('WH1_F16_electric_PDRSAug24__storage_volume', period)
+      print("storage_volume",storage_volume)
```
Comment (alisha17) [inline openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py:175]: can you remove this?

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml (hunk 1 above)
Comment (SelvaRC) [reply openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml]: i have updated this on a new branch, pushed on (PR 441)

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py (hunk 2 above)
Comment (SelvaRC) [reply openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py]: Done

File: openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py (hunk 4 above)
Comment (SelvaRC) [reply openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/activity_eligibility/WH1_F16_electric_PDRSAug24_activity_eligibility_parameters.py:175]: Done

File: openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml (hunk 3 above)
Comment (SelvaRC) [reply openfisca_nsw_safeguard/tests/ESS_PDRS_Estimator/WH1_F16_electric_PDRSAug24/WH1_F16_electric_PDRSAug24_installation_final_activity_eligibility.yaml:273]: Done

**Review by k8mckay — APPROVED** [review]
Note: uppercase fixed in PR 441

**Pull request merged**